docs/docs/installation/configuring-superset.mdx

```diff
@@ -71,9 +71,11 @@ WTF_CSRF_EXEMPT_LIST = [‘’]
 
 ### Using a production metastore
 
-By default Superset is configured to use SQLite, it's a simple and fast way to get you started
-(no installation needed). But for production environments you should use a different database engine on
-a separate host or container.
+By default, Superset is configured to use SQLite, which is a simple and fast way to get started
+(without requiring any installation). However, for production environments,
+using SQLite is highly discouraged due to security, scalability, and data integrity reasons.
+It's important to use only the supported database engines and consider using a different
+database engine on a separate host or container.
```

Comment:
```suggestion
We recommend using another supported database engine on a separate host or container.
```

Reply:
shorter but seems a bit confusing to say "another supported database engine on a separate host"